cluster/manifests/skipper/deployment.yaml

```diff
@@ -184,3 +184,11 @@ spec:
             name: skipper-default-filters
             optional: true
 {{ end }}
+{{ if eq .ConfigItems.enable_dedicate_nodepool_skipper "true"}}
```

Comment:
The config item should be defined as `enable_dedicate_nodepool_skipper: "false"` in the default-configs file.

Reply:
fixed